libraries/tlv-account-resolution/src/state.rs

```diff
@@ -1081,13 +1254,9 @@ mod tests {
         // CPI account infos should have the instruction account infos
         // and the extra required account infos from the validation account,
         // and they should be in the correct order.
-        let mut all_account_infos = ix_account_infos.to_vec();
-        all_account_infos.extend(extra_account_infos.iter().cloned());
-        all_account_infos.push(extra_pda_info1);
-        all_account_infos.push(extra_pda_info2);
-
-        assert_eq!(cpi_account_infos.len(), all_account_infos.len());
-        for (a, b) in std::iter::zip(cpi_account_infos, all_account_infos) {
+        let check_account_infos = &all_account_infos[..9];
```

Comment:
Sorry, I think I'm missing something here, why are we not checking all of the account infos here?

Reply:
I can modify the comment above this line, but you can see the last two account infos added are labeled as `arb` something

Reply:
Gotcha, I see what you mean now -- that part was definitely not clear to me. Feel free to keep this as is